### DISCUSSION TITLE
Install issue

### DISCUSSION THREAD

**Discussion by metricspace64** [nzbgetcom/nzbget#318] [General]
==> ERROR: Failure while downloading https://github.com/nzbgetcom/nzbget/releases/download/testing/nzbget-24.2-testing-20240717-amd64.deb
    Aborting...

Is this not the correct url?

**Comment by dnzbk** [comment]
Outdated. Please use [this](https://github.com/nzbgetcom/nzbget/releases/download/testing/nzbget-24.2-testing-20240719-amd64.deb) one.

**Comment by dnzbk** [comment]
@metricspace64 
[Here](https://github.com/nzbgetcom/nzbget/releases) can be found all the latest actual links to testing and stable releases.
